src/interfaces.ts

```diff
@@ -70,6 +74,8 @@ export interface IUserService {
         userId: string,
         token: UserExpoPushTokenRequestBody
     ) => Promise<User>;
+
+    pushNotification: (message: string) => Promise<void>;
```

Comment:
Arrow functions as class methods don't provide any advantage unless you plan to assign the method to a variable in another context. Leaving them as classic method definitions makes their intent clearer, IMO. They are methods and not variables, after all.
```suggestion
    public pushNotification(message: string): Promise<void>;
```

Reply:
Copilot tell me to do that, haha